BICCN-policies/DRAFT Citation and identifier policy.md

```diff
@@ -0,0 +1,49 @@
+# Policy regarding support of data and resource citation and use of persistent identifiers
+
+## For data submitters/authors
+
+### Persistent identifiers + metadata for BICCN datasets
+- BICCN consortium members will submit their data to an approved BRAIN repository for publication.  
+- Datasets must include a standard set of metadata specified by the BICCN (TODO:  Get list from Carol)
```

Comment:
Can link to Dataset metadata template which is in the BICCN/BCDC Metadata repository.